lib/Controller/SettingsController.php

```diff
@@ -39,7 +39,7 @@ public function __construct(
 	/**
 	 * Update user setting
 	 *
-	 * @param 'attachment_folder'|'read_status_privacy'|'typing_privacy'|'play_sounds' $key Key to update
+	 * @param 'attachment_folder'|'read_status_privacy'|'typing_privacy'|'play_sounds'| 'blur_background' $key Key to update
```

Comment:
Don't add new things to old APIs :-X
```suggestion
	 * @param 'attachment_folder'|'read_status_privacy'|'typing_privacy'|'play_sounds' $key Key to update
```